hr_attendance_geolocation/static/src/js/attendance_geolocation.js

```diff
@@ -106,16 +109,15 @@ odoo.define("hr_attendance_geolocation.attendances_geolocation", function (requi
                 );
                 pinBoxVal = this.$(".o_hr_attendance_PINbox").val();
             }
+            var ctx = _.extend({}, session.user_context, {
+                latitude: position.coords.latitude,
+                longitude: position.coords.longitude,
+            });
```

Comment:
Better to go with vanilla JS and avoid underscore as much as possible
```suggestion
            const ctx = Object.assign(session.user_context, {
                latitude: position.coords.latitude,
                longitude: position.coords.longitude,
            });
```